### ISSUE TITLE
Jasmine is timing out in protractor, Selenium Server crashes afterwards (issue is present in Chrome, Firefox - OK)

### ISSUE THREAD

**Issue by iljapavlovs** [angular/protractor#1309]
Hi! Really need some help on this one, I have been struggling for a really long time and cant figure it out on my own!
I am running tests in chrome and when protractor logs in to the system, it suddenly stops and , basically, crashes - 
- at first Jasmine is timing out (explicitly extended the timeout), 
- and then protractor cant quit the browser for long time (Log from Selenium Server -`Timed out receiving message from renderer: 295.458,  Timed out receiving message from renderer: 10.000`)

So here is the code:

``` javascript
describe('Login with username and password', function () {
        it('Logging in with VALID credentials', function () {
            browser.get('');
            expect(browser.getCurrentUrl()).toContain('login');
            element(by.id('username')).sendKeys("admin");
            element(by.id('password')).sendKeys("password");
            element(by.css('input[type="submit"]')).click();//here I am login into the system. On successful login, system loads lots of resources into the browser for subsequent bus logic
           // browser.waitForAngular();
            expect(browser.getCurrentUrl()).toContain('main');
            var greeting = element(by.css("div.greeting.ng-binding.ng-scope"));
            expect(greeting.getText()).toEqual('Hei, admin');
            element(by.css('.logout')).click();
        });
    });
```

Log from protractor when it fails to quit browser and/or Selenium Server.

```
C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\atoms\error.js:109
  var template = new Error(this.message);
                 ^
ScriptTimeoutError: timeout: Timed out receiving message from renderer: 10.000
  (Session info: chrome=37.0.2062.103)
  (Driver info: chromedriver=2.10.267521,platform=Windows NT 6.1 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 310.28 seconds
Build info: version: '2.42.2', revision: '6a6995d', time: '2014-06-03 17:42:03'
System info: host: 'LVALTP1065', ip: '192.168.56.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.7.0_67'
Session ID: 98ef9389e06f7736b7587ecd362729c0
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{platform=XP, acceptSslCerts=true, javascriptEnabled=true, browserName=chrome, chrome={userDataDir=C:\Users\ILJA~1.PAV\AppData\Local\Temp\scoped_dir3800_4224}, rotatable
=false, locationContextEnabled=true, version=37.0.2062.103, takesHeapSnapshot=true, cssSelectorsEnabled=true, databaseEnabled=false, handlesAlerts=true, browserConnectionEnabled=false
, nativeEvents=true, webStorageEnabled=true, applicationCacheEnabled=false, takesScreenshot=true}]
    at new bot.Error (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\atoms\error.js:109:18)
    at Object.bot.response.checkResponse (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\atoms\response.js:106:9)
    at C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\webdriver.js:277:20
    at C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\goog\base.js:1243:15
    at webdriver.promise.ControlFlow.runInNewFrame_ (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:1539:20
)
    at notify (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:362:12)
    at notifyAll (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:331:7)
    at resolve (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:309:7)
    at fulfill (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:429:5)
    at C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:1406:10
==== async task ====
WebDriver.executeScript()
    at webdriver.WebDriver.schedule (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\webdriver.js:268:15)
    at webdriver.WebDriver.executeAsyncScript (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\webdriver.js:496:15)
    at Protractor.waitForAngular (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\lib\protractor.js:913:22)
    at to.(anonymous function) [as getCurrentUrl] (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\lib\protractor.js:56:7)
    at null.<anonymous> (C:\Users\ilja.pavlovs\Desktop\AngularApp\src\e2e\specs\test.js:28:28)
    at C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd\index.js:94:14
    at webdriver.promise.ControlFlow.runInNewFrame_ (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:1539:20
)
    at webdriver.promise.ControlFlow.runEventLoop_ (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:1404:8)
    at wrapper [as _onTimeout] (timers.js:261:14)
==== async task ====
WebDriver.quit()
    at webdriver.WebDriver.schedule (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\webdriver.js:268:15)
    at webdriver.WebDriver.quit (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\webdriver.js:333:21)
    at HostedDriverProvider.teardownEnv (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\lib\driverProviders\hosted.js:47:16)
    at driverprovider_.setupEnv.then.then.then.then.passed (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\lib\runner.js:283:35)
    at _fulfilled (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\q\q.js:797:54)
    at self.promiseDispatch.done (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\q\q.js:826:30)
    at Promise.promise.promiseDispatch (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\q\q.js:759:13)
    at C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\q\q.js:573:44
    at flush (C:\Users\ilja.pavlovs\AppData\Roaming\npm\node_modules\protractor\node_modules\q\q.js:108:17)
```

 So basically it crashes at `expect(browser.getCurrentUrl()).toContain('main');` - it was line 28 originally.
In Firefox all works perfectly! I have tried:
- extending Jasmine timeout
- using Chromedriver without selenium server
- running wthout browser.waitForAngular()
- with browser.waitForAngular() uncommented;
- with all kind of different settings
- protractor 1.2.0 and 1.1.1 versions

I would very appreciate any help with this one!

**Comment by losttime** [comment]
I've found the downgrading to Java 7 is an effective workaround for local development.

I just change the JAVA_HOME to the location of Java 7 and open a new shell to run my tests from.
